### PR TITLE
Remove unused code

### PR DESCRIPTION
**Pull request by insacuri**
# Motivation and Context

# What has changed

# How to test?

# Links

### CODE REVIEW

File: src/main/java/uk/gov/ons/ctp/response/lib/common/time/DateTimeUtil.java [hunk 1]
```diff
@@ -1,25 +1,12 @@
 package uk.gov.ons.ctp.response.lib.common.time;
```
Comment (akssinhaamit) [inline src/main/java/uk/gov/ons/ctp/response/lib/common/time/DateTimeUtil.java:1]: Is this not being used by schedulers?

File: src/main/resources/application.yml [hunk 2]
```diff
@@ -88,18 +88,6 @@ data-grid:
 
   report-lock-time-to-live-seconds: 300
 
-# the thread that sends actions to handlers
```
Comment (akssinhaamit) [inline src/main/resources/application.yml:91]: Is it not being used by scheduler?

**Review by akssinhaamit — CHANGES_REQUESTED** [review]
`ActionRequestValidator.java` can be removed I believe.

File: src/main/resources/application.yml (hunk 2 above)
Comment (insacuri) [reply src/main/resources/application.yml:91]: That was used by the old version of the scheduler (with redis and the @scheduled annotation).  The new one has its timing set in the helm chart

File: src/main/java/uk/gov/ons/ctp/response/lib/common/time/DateTimeUtil.java (hunk 1 above)
Comment (insacuri) [reply src/main/java/uk/gov/ons/ctp/response/lib/common/time/DateTimeUtil.java:1]: I couldn't find anything in the codebase that was using the 2 functions I deleted.  Can you see anywhere that it's used?

**Review by akssinhaamit — APPROVED** [review]

**Pull request merged**